ui/app/ducks/alerts/unconnected-account.js

```diff
@@ -94,7 +94,7 @@ export const dismissAndDisableAlert = () => {
   return async (dispatch) => {
     try {
       await dispatch(disableAlertRequested())
-      await dispatch(setAlertEnabledness(name), false)
+      await dispatch(setAlertEnabledness(name, false))
```

Comment:
I didn't intend to commit this here - this has been fixed in a separate PR (#8561)

Reply:
#8561 has been merged, and this PR has been rebased on `develop`, so this should be gone from the diff now.